jwala-common/src/main/java/com/cerner/jwala/common/properties/PropertyKeys.java

```diff
@@ -27,7 +27,8 @@
     JSCH_CHANNEL_SHELL_READ_INPUT_SLEEP_DURATION("jsch.channel.shell.read.input.sleep.duration"),
     JSCH_EXEC_READ_REMOTE_OUTPUT_LOOP_SLEEP_TIME("jsch.exec.read.remote.output.loop.sleep.time"),
     JSCH_SHELL_READ_REMOTE_OUTPUT_TIMEOUT("jsch.shell.read.remote.output.timeout"),
-    JSCH_EXEC_READ_REMOTE_OUTPUT_TIMEOUT("jsch.exec.read.remote.output.timeout");
+    JSCH_EXEC_READ_REMOTE_OUTPUT_TIMEOUT("jsch.exec.read.remote.output.timeout"),
+    MAX_JWALA_HISTORY("max.jwala.history");
```

Comment:
Can we standardize that all of the properties specifically for jwala start with jwala?
So this property key would change to "jwala.max.history".